verde/coordinates.py

```diff
@@ -8,7 +8,7 @@
 from .utils import kdtree
 
 
-def check_region(region):
+def check_region(region, latlon=False):
```

Comment:
Would `geographic=False` be a better name? We already say "geographic" in the docs and comments to mean latitude and longitude.

Reply:
I like `geographic` more than the `latlon`. It improves readability and will be in agreement with the docs.